YSI_Coding/y_unique/y_unique_entry.inc

```diff
@@ -141,6 +141,10 @@ Optional plugins:
 #endif
 
 #if defined YSI_LOCK_MODE
+	#if !defined YSI_gLockData
+	    new
+			YSI_gLockData[21];
+	#endif
```

Comment:
This error is on purpose.

Reply:
I removed it. What is the reason of giving the error?

Reply:
Because it is hidden behind `YSI_LOCK_MODE`.  I'm not really sure how you triggered the error, but it shouldn't happen if things are included in the correct order.  If there is an order in which the error is triggered, that is not the correct fix - resolving the include order correctly is.

Reply:
When I include 
```Pawn
#include <YSI_Server\y_lock>
```
at the very top (after `a_samp.inc` and before any YSI library), I receive
```Pawn
YSI_Core\y_core\y_utils_impl.inc(222) : error 017: undefined symbol "UNIQUE_SYMBOL"
```
but if I include `y_lock` after another YSI library, I receive
```Pawn
YSI_Coding\y_unique\y_unique_entry.inc(152) : error 017: undefined symbol "YSI_gLockData"
```